res/openvpn-gui-res-it.rc

```diff
@@ -411,8 +448,30 @@ BEGIN
     IDS_ERR_DISCONN_SCRIPT_TIMEOUT "Il valore di registro ""disconnectscript_timeout"" deve essere un numero tra 1 and 99."
     IDS_ERR_PRECONN_SCRIPT_TIMEOUT "Il valore di registro ""preconnectscript_timeout"" deve essere un numero tra 1 and 99."
     IDS_ERR_CREATE_REG_KEY "Errore nella creazione della chiave HKLM\\SOFTWARE\\OpenVPN-GUI."
-    IDS_ERR_OPEN_WRITE_REG "Fallita l'apertura del registro in scrittura. Puoi far partire questa applicazione " \
+    IDS_ERR_OPEN_WRITE_REG "Errore nell'apertura del registro in scrittura. Puoi far partire questa applicazione " \
                            "una volta che l'amministratore ha aggiornato il registro."
-    IDS_ERR_READ_SET_KEY "Errore nella lettura e settaggio della chiave di registro ""%s""."
+    IDS_ERR_READ_SET_KEY "Errore nella lettura e modifica della chiave di registro ""%s""."
     IDS_ERR_WRITE_REGVALUE "Errore nella scrittura del valore di registro ""HKEY_CURRENT_USER\\%s\\%s""."
+    
+    /* importation */
+    IDS_ERR_IMPORT_EXISTS "Esiste già una configurazione con il nome ""%s"" already exists."
+    IDS_ERR_IMPORT_FAILED "Errore nell'importazione del file. Non ho potuto creare il percorso seguente.\n\n" \
+                          "%s\n\nControlla di avere i permessi giusti."
+    IDS_NFO_IMPORT_SUCCESS "File importato con successo."
+
+    /* save/delete password */
+    IDS_NFO_DELETE_PASS "Premi OK per cancellare le password salvate per la configurazione ""%s"""
+
+    /* Token password related */
+    IDS_NFO_TOKEN_PASSWORD_CAPTION "OpenVPN - Password Token"
+    IDS_NFO_TOKEN_PASSWORD_REQUEST "Inserisci la Password/PIN per il token '%S'"
+
+    IDS_NFO_AUTH_PASS_RETRY "Credenziali errate. Riprova..."
+    IDS_NFO_KEY_PASS_RETRY  "Password errata. Riprova..."
+    IDS_ERR_INVALID_PASSWORD_INPUT "Carattere non valido nella password"
+    IDS_ERR_INVALID_USERNAME_INPUT "Carattere non valido nel nome utente"
+    IDS_NFO_AUTO_CONNECT    "Connessione automatica tra %u secondi..."
+    IDS_NFO_CLICK_HERE_TO_START "L'interfaccia OpenVPN è già in esecuzione. Clicca con il tasto destro sull'icona della tray per cominciare."
```

Comment:
instead of "icona della tray", how about "icona sulla barra delle applicazioni"? I believe (but I am not sure, so please correct me) that this is how that area is called in Italian (it may have changed with the latest windows releases). O "nell'area di notifica", but I think the first is more understandable.

Reply:
Yeah, that would be "area di notifica". I'll change it.